### ISSUE TITLE
"Guest" can't be translated

### ISSUE THREAD

**Issue by Studio384** [GetLuna/Luna#1281]
It is impossible to translate "Guest" as the new user after installation.

**Comment by ninjagolloyd** [comment]
Huh. Is there a string to translate it?

**Comment by Studio384** [comment]
Yes, the installer pulls it to name the account, but this, or course, requires it to already be translated by then. Changing the boards default language should update the guest's name. But perhaps even better would be to allow admins to change the name of the guest account in the Backstage. Might be nice for some forums.

**Comment by ninjagolloyd** [comment]
I'm no expert but
``` php
define('guestusrname', '*');
```
